adapters/dmx/dmx.go

```diff
@@ -257,6 +263,7 @@ func (adapter *DmxAdapter) MakeBids(request *openrtb.BidRequest, externalRequest
 
 func fetchParams(params dmxExt, inst openrtb.Imp, ins openrtb.Imp, imps []openrtb.Imp, banner *openrtb.Banner, video *openrtb.Video, intVal int8) []openrtb.Imp {
 	var tempimp openrtb.Imp
+	var checkValue *uint64
 	tempimp = inst
```

Comment:
While we're here, this assignment isn't necessary. `inst` is already a copy of the imp since it's passed by value. Please consider using `inst` directly.

Reply:
Forgot it was a shared in-memory